addons/popochiu/editor/main_dock/tab_room.gd

```diff
@@ -2,7 +2,7 @@
 extends VBoxContainer
 ## Handles the Room tab in Popochiu's dock
 
-const PopochiuObjectRow := preload('object_row/popochiu_object_row.gd')
+const PopochiuObjectRow := preload('res://addons/popochiu/editor/main_dock/object_row/popochiu_object_row.gd')
```

Comment:
The reason I changed that was because I was using vscode to edit code and it was throwing warnings throughout the code.

When I changed from a relative path to the absolute path vscode stopped showing the warnings.